.github/workflows/fuzzydata-test.yml

```diff
@@ -51,7 +51,7 @@ jobs:
           conda info
           conda list
       - name: test-fuzzydata (engine ${{matrix.engine}}, python ${{matrix.python-version}})
-        run: python -m pytest modin/experimental/fuzzydata/test/test_fuzzydata.py -Wignore::UserWarning
+        run: python -m pytest modin/experimental/fuzzydata/test/test_fuzzydata.py -Wignore::UserWarning --log-file=/tmp/fuzzydata-test-wf-${{matrix.engine}}/run.log --log-file-level=DEBUG
```

Comment:
Wouldn't `INFO` level be enough for us?

Reply:
Good call, I can modify `fuzzydata` to output all pertinent information to `INFO`.

Reply:
> Wouldn't INFO level be enough for us?

@mvashishtha what do you think?

Reply:
I don't think the modin logs are useful at all here, so we should set the level to whatever @suhailrehman thinks will get enough info from fuzzydata